tests/functional/config/nix-with-bang-include.conf

```diff
@@ -0,0 +1,2 @@
+experimental-features = nix-command
```

Comment:
I would do a test like 
```
system = asdf
```
because `experimental-features` is much more likely to interfere with the test suite itself